app/mailers/mailings/devise.rb

```diff
@@ -5,35 +5,47 @@ class Devise < ApplicationMailer
     include ::Devise::Controllers::UrlHelpers
 
     def confirmation_instructions(record, token, _opts = {})
-      Rails.logger.debug(confirmation_url(record, confirmation_token: token)) if Rails.env.development?
+      Rails.logger.debug("Sending confirmation instructions")
```

Comment:
This log message will likely not help finding the (potential) problem that occurs with the occasionally wrong activation link as it lacks context (who has just received the confirmation instructions? Was this a new member? Was it the first email the received? Logging the user id or email allows for searching for clues in the Koala logs and Mailgun history)